DataFormats/Scouting/interface/Run3ScoutingElectron.h

```diff
@@ -97,6 +99,7 @@ class Run3ScoutingElectron {
   float sMaj() const { return sMaj_; }
   unsigned int seedId() const { return seedId_; }
   std::vector<float> const& energyMatrix() const { return energyMatrix_; }
+  std::vector<unsigned int> const& detIds() const { return detIds_; }
```

Comment:
I'm wondering if this type should match exactly the one of `DetId::id_` which is [`uint32_t`](https://github.com/cms-sw/cmssw/blob/fec38251c98859243b634021276df948344f81de/DataFormats/DetId/interface/DetId.h#L53). The two may be equivalent in most cases, but maybe better to just use `uint32_t` here?

Reply:
Ah, I mistakenly thought this was a ROOT-defined data type. Didn't realize that it is C++ implicit. Will do!